### PR TITLE
Adding missing unit_currency and entity_type schema fields

### PR DESCRIPTION
**Pull request by ITSS-official**
Hello,

We added the different fields to the schema.

For the commerce_stock_transaction we only add the field unit_currency, but like unit_cost is null.

We commented the line to populate it with real value into the modules/field/src/Plugin/Field/FieldType/StockLevel.php file.

For the commerce_stock_location_level table, we add the entity_type field too and we populate it.

But we would like to know how to test when the cron add data to this table.

Sincerely,

itss-official team

### CODE REVIEW

**Comment by BBGuy** [comment]
Thanks. I will review this PR as soon as I have time.

File: modules/local_storage/commerce_stock_local.install [hunk 1]
```diff
@@ -98,6 +98,12 @@ function commerce_stock_local_schema() {
         'precision' => 19,
         'scale' => 6,
       ],
+      'unit_currency' => [
```
Comment (BBGuy) [inline modules/local_storage/commerce_stock_local.install]: Looking at commerce currency_code the cost should be a varchar(3): 
   'currency_code' => [
          'description' => 'The currency code.',
          'type' => 'varchar',
          'length' => 3,
        ],

File: src/Plugin/StockEvents/CoreStockEvents.php [hunk 2]
```diff
@@ -53,7 +53,7 @@ public function stockEvent(Context $context, PurchasableEntityInterface $entity,
       ->getService($entity);
     // Use the stock updater to create the transaction.
     $transaction_id = $stockService->getStockUpdater()
-      ->createTransaction($entity, $location->getId(), '', $quantity, NULL, $transaction_type, $metadata);
+       ->createTransaction($entity, $location->getId(), '', $quantity, NULL, $unit_currency = NULL, $transaction_type, $metadata);
```
Comment (BBGuy) [inline src/Plugin/StockEvents/CoreStockEvents.php]: remove extra space

**Comment by BBGuy** [comment]
Great work. |I found one issue to do with the definition of the unit cost https://github.com/BBGuy/commerce_stock/pull/56/files#r218089184 once fixed I can finish testing and commit.

**Comment by ITSS-official** [comment]
> Great work. |I found one issue to do with the definition of the unit cost https://github.com/BBGuy/commerce_stock/pull/56/files#r218089184 once fixed I can finish testing and commit.

Perfect. We will see issue ASAP.

**Comment by ITSS-official** [comment]
Hello BBGuy,

We had committed the new changes last week: "Changing unit_currency definition to currency_code."

**Comment by BBGuy** [comment]
Supper great work. I committed this changes + a few small changes I came across while testing

**Pull request closed without merging**